src/vs/workbench/contrib/workspace/browser/workspace.contribution.ts

```diff
@@ -441,6 +447,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 					localize('workspace.trust.startupPrompt.once', "Ask for trust the first time an untrusted workspace is opened."),
 					localize('workspace.trust.startupPrompt.never', "Do not ask for trust when an untrusted workspace is opened."),
 				]
+			},
+			[WORKSPACE_TRUST_EMPTY_WORKSPACE]: {
+				type: 'boolean',
+				default: false,
```

Comment:
one experience I was thinking about is the brand new user experience. they will open a new window and have a restricted mode banner and status bar entry. perhaps this is justification to use true as the default, or alternately, we wait until any files are opened to show the banner

Reply:
That is something that we can adjust based on user feedback.